### PR TITLE
Fwmt 428 last updated added to jobs table

### PR DESCRIPTION
**Pull request by ChrisHardmanONS**
added changes to data base through liquibase v18, also added new column to job dto and entity

### CODE REVIEW

**Review by soundargit — CHANGES_REQUESTED** [review]
add test and fix build

**Review by ChrisHardmanONS — COMMENTED** [review]
done

File: src/integrationTest/java/uk/gov/ons/fwmt/resource_service/controller/JobControllerIT.java [hunk 1]
```diff
@@ -33,7 +31,7 @@
 @Transactional
 public class JobControllerIT {
```
Comment (soundargit) [inline src/integrationTest/java/uk/gov/ons/fwmt/resource_service/controller/JobControllerIT.java:32]: Check/expect for the values  in all rest operation test.

File: src/test/java/uk/gov/ons/fwmt/resource_service/controller/JobControllerTest.java [hunk 2]
```diff
@@ -18,24 +18,21 @@
...
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
 
 @RunWith(MockitoJUnitRunner.class)
 public class JobControllerTest {
```
Comment (soundargit) [inline src/test/java/uk/gov/ons/fwmt/resource_service/controller/JobControllerTest.java:33]: add the expected value in all respective tests

**Review by soundargit — CHANGES_REQUESTED** [review]

File: src/integrationTest/java/uk/gov/ons/fwmt/resource_service/controller/JobControllerIT.java [hunk 3]
```diff
@@ -42,17 +40,21 @@ public void getJobByJobIdIT() throws Exception {
...
 
   @Test
   public void updateJobIT() throws Exception {
     TMJobEntity jobEntity = new TMJobEntity();
     jobEntity.setTmJobId("1234-5678");
     jobEntity.setLastAuthNo("1234");
+    jobEntity.setLastUpdated(LocalDateTime.parse("2018-08-01T01:06:01",DateTimeFormatter.ISO_LOCAL_DATE_TIME));
     jobRepo.save(jobEntity);
     mockMvc.perform(
         (put("/jobs")).contentType(MediaType.APPLICATION_JSON).content(JOB_JSON).with(httpBasic("user", "password")));
```
Comment (soundargit) [inline src/integrationTest/java/uk/gov/ons/fwmt/resource_service/controller/JobControllerIT.java:60]: add the expected in updateJobIT

**Review by soundargit — CHANGES_REQUESTED** [review]

**Review by soundargit — APPROVED** [review]

**Pull request merged**